schema/schema.go

```diff
@@ -68,31 +67,23 @@ type StateTree struct {
 // Base64EncodedBytes is a base64-encoded binary value.
 type Base64EncodedBytes []byte
 
-// ChainHead represents a head tipset.
-type ChainHead []cid.Cid
-
-// PreconditionsBlockSeq are the preconditions for a blockseq vector.
-type PreconditionsBlockSeq struct {
-	GenesisTs time.Time  `json:"genesis_ts,omitempty"`
-	ChainHead *ChainHead `json:"chain_head,omitempty"`
-}
-
 // Preconditions contain the environment that needs to be set before the
 // vector's applies are applied.
 type Preconditions struct {
-	*PreconditionsBlockSeq
-
 	// Epoch must be interpreted by the driver as an abi.ChainEpoch in Lotus, or
 	// equivalent type in other implementations.
 	Epoch     int64      `json:"epoch,omitempty"`
 	StateTree *StateTree `json:"state_tree,omitempty"`
 
+	// BaseFee is an optional base fee to inject into the VM when feeding this
+	// message. If absent, it defaults to 100 attoFIL.
+	BaseFee *big.Int `json:"basefee,omitempty"`
```

Comment:
Maybe `base_fee`, since the field is called `BaseFee`.

Reply:
We called it `base_fee` in the `TipSet` type, so just keeping this for consistency.